src/jest.timezone.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+// Setting the timezone to UTC to ensure same timezone for CI and local environments
+module.exports = async () => {
+  process.env.TZ = "UTC";
```

Comment:
Unit tests should have mocked date and timezone. Just to make them more deterministic. 
Would be great to have global test setup (in unit tests) which sets current time to `2015-10-21T07:28:00Z` and timezone to `UTC` of course. This would remove complete set of tests failing for a wrong reason.

Selection of "current time" fake is very important. It needs to be:
- easy to remember
- easy to distinct

In general, we should keep all dates UTC and convert only when showing to user. Common mistake is to convert dates too early. That makes a tricky topic a lot simpler.

Reply:
Unit tests should have mocked date and timezone. Just to make them more deterministic. 
Would be great to have global test setup (in unit tests) which sets current time to `2015-10-21T07:28:00Z` and timezone to `UTC` of course. This would remove complete set of tests failing for a wrong reason.

Selection of "current time" fake is very important. It needs to be:
- easy to remember
- easy to distinct

In general, we should keep all dates UTC and convert only when showing to user. Common mistake is to convert dates too early. That makes a tricky topic a lot simpler.